assets/src/dashboard/animations/animator/animator.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { GeneralAnimationPropTypes } from './types';
+
+function Animator({ id, animation, config, trigger }) {
```

Comment:
How is Animator different from Animation?

Reply:
`Animation` excepts keyframes and does _not_ reference another 'animation'.  

`Animator` does _not_ allow keyframes, it only accepts an 'animation' reference (by string name), and it expects an array of configurations including the selector (which determines which elements to animate).